pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -525,3 +526,142 @@ func (d *linuxDataplane) CleanUpNamespace(args *skel.CmdArgs) error {
 
 	return nil
 }
+
+func (d *linuxDataplane) CheckNetworking(args *skel.CmdArgs, prevResult *current.Result) error {
+	caliVethName := "cali" + args.ContainerID[:utils.Min(11, len(args.ContainerID))]
+	err := d.configureInterface(caliVethName, prevResult)
+	if err != nil {
+		d.logger.WithError(err).Warnf("Failed to configure interface %s", caliVethName)
+	}
+
+	exists, err := interfaceExistsInProcSys(caliVethName)
+	if err != nil {
+		d.logger.WithError(err).Warnf("Failed to check if %s interface exists", caliVethName)
+	}
+
+	if !exists {
+		return fmt.Errorf("Interface %s should exist but does not", caliVethName)
+	}
+
+	// TODO: Need to check the state of the interface specified in the prevResult of the args as per spec
+	return nil
+}
+
+func (d *linuxDataplane) configureInterface(name string, prevResult *current.Result) error {
+	// TODO: Do we need something to check for active interfaces? Something similar to the below?
```

Comment:
We shouldn't need this since this is only called by CHECK (which should only be called on active interfaces).